src/Tizen.NUI/src/internal/XamlBinding/PositionTypeConverter.cs

```diff
@@ -59,6 +59,12 @@ public override object ConvertFromInvariantString(string value)
                     int z = (int)GraphicsTypeManager.Instance.ConvertScriptToPixel(parts[2].Trim());
                     return new Position(x, y, z);
                 }
+                else if (parts.Length == 2)
```

Comment:
Position and Size gets float typed parameter. and as I know xaml support float type. (maybe we can use Single class)
but this looks like converted and int type. 
how do you think about?

Reply:
Hello, Mr. SSong, You're right. Thank you for your remind.